client/ayon_core/hosts/max/api/lib.py

```diff
@@ -221,41 +226,31 @@ def reset_scene_resolution():
     scene resolution can be overwritten by a folder if the folder.attrib
     contains any information regarding scene resolution.
     """
-
-    folder_entity = get_current_project_folder(
-        fields={"attrib.resolutionWidth", "attrib.resolutionHeight"}
-    )
-    folder_attributes = folder_entity["attrib"]
-    width = int(folder_attributes["resolutionWidth"])
-    height = int(folder_attributes["resolutionHeight"])
+    task_entity = get_current_task_entity()
+    task_attributes = task_entity["attrib"]
+    width = int(task_attributes["resolutionWidth"])
+    height = int(task_attributes["resolutionHeight"])
 
     set_scene_resolution(width, height)
 
 
-def get_frame_range(folder_entiy=None) -> Union[Dict[str, Any], None]:
-    """Get the current folder frame range and handles.
+def get_frame_range(task_entity=None) -> Union[Dict[str, Any], None]:
+    """Get the current task frame range and handles
 
     Args:
-        folder_entiy (dict): Folder eneity.
+        task_entity (dict): Task Entity.
 
     Returns:
         dict: with frame start, frame end, handle start, handle end.
     """
     # Set frame start/end
-    if folder_entiy is None:
-        folder_entiy = get_current_project_folder()
-
-    folder_attributes = folder_entiy["attrib"]
-    frame_start = folder_attributes.get("frameStart")
-    frame_end = folder_attributes.get("frameEnd")
-
-    if frame_start is None or frame_end is None:
-        return {}
-
-    frame_start = int(frame_start)
-    frame_end = int(frame_end)
-    handle_start = int(folder_attributes.get("handleStart", 0))
-    handle_end = int(folder_attributes.get("handleEnd", 0))
+    if task_entity is None:
+        task_entity = get_current_task_entity()
```

Comment:
This can now use the implementation added by this PR: https://github.com/ynput/ayon-core/pull/353